services/src/main/java/org/keycloak/services/resources/LoginActionsService.java

```diff
@@ -1050,7 +1052,10 @@ public void ignore() {
         } else if (context.getStatus() == RequiredActionContext.Status.FAILURE) {
             response = interruptionResponse(context, authSession, action, Error.CONSENT_DENIED);
         } else {
-            throw new RuntimeException("Unreachable");
+            if (!Objects.equals(factory.getId(), DeleteAccount.PROVIDER_ID)) {
+                throw new RuntimeException("Unreachable");
+            }
+            response = Response.status(302).location(Urls.accountPage(session.getContext().getUri().getBaseUri(), realm.getName())).build();
```

Comment:
A more general comment. I wonder, should we redirect to some meaningful page with something like "Yay, your account was deleted!" instead of redirecting back to the account console?

Reply:
@ssilvert Yes but in order to display the login screen, we need a client. And which client that should be? ;)

I guess the only plausible solution is to really just display some static page (in the design of login page) with some informative message.

Reply:
@vmuzikar  @zak905 I think it will still work.  The client isn't deleted.  Only the user account is deleted.  So if you redirected to auth/realms/<realm name>/account/#personal-info you would get the login screen.

Reply:
@ssilvert Yes but we don't want to redirect back to the Account Console, right? Even if it only displays a login screen. I think an informative message about successful account deletion would be more useful.

Reply:
jFTR, this topic is still valid. :)

Reply:
This should be handled in the DeleteAction, not in LoginActionsService. The DeleteAction should probably reset the flow, cleanup the session/cookies, etc. Then just respond with context.challenge(..) with a page stating the account has been deleted.

@mposolda u know the auth flows better than me, do you have suggestions here?

Reply:
@stianst I've commented [above](https://github.com/keycloak/keycloak/pull/7176/files#r489235064) in relation to this.